readme.md

```diff
@@ -44,6 +52,43 @@ Helpful tools for students with engineering background
 - [Wolfram Mathematica](https://www.wolfram.com/mathematica/pricing/students-individuals.php) — tools for technical computing. *Not completly free, but with high discount.*
 
 
+## Education
+
+Platforms for education
+
+- [Invision](https://www.invisionapp.com/education-signup) — the digital product design platform powering the world’s best user experiences
```

Comment:
Don't understand why Invision, which is primarily used for design purposes, belongs to Education secion